client/my-sites/themes/theme-options.js

```diff
@@ -30,9 +30,8 @@ import {
 
 const identity = ( theme ) => theme;
 
-function getAllThemeOptions( { translate, blockEditorSettings } ) {
-	const isFSEActive = blockEditorSettings?.is_fse_active ?? false;
-
+function getAllThemeOptions( { translate, activeThemeData } ) {
+	const isFSEActive = activeThemeData?.[ 0 ]?.theme_supports[ 'block-templates' ] ?? false;
```

Comment:
Since we have this same check (`activeThemeData?.[ 0 ]?.theme_supports[ 'block-templates' ]`), maybe there's a good way to abstract it and return just whether or not FSE is active, so we don't it have to write it out every time?

Reply:
Good idea! I will create a new function inside a lib file to return the FSE is active value using the query data.